### ISSUE TITLE
Documentation & error message about graphviz dependency

### ISSUE THREAD

**Issue by wbeard** [DaveWM/willa#8]
Hey there! My co-worker recommended your library to me and I was reading through your [blog post](https://engineering.fundingcircle.com/blog/2019/08/27/kafka-streams-the-clojure-way/). 

It looks great!

When I ran `(wv/view-topology topology)`, the following exception was thrown:
```
Execution error (IOException) at java.lang.UNIXProcess/forkAndExec (UNIXProcess.java:-2). error=2, No such file or directory
```

After some digging, it looks like [rhizome](https://github.com/ztellman/rhizome) has a [dependency on graphviz](https://github.com/ztellman/rhizome#usage), which has to be installed by an os package manager. Installing graphviz solves the issue.

I think it would be helpful to have some documentation noting the need to install the dependency to use this method & maybe a `try` around the usage of `rhizome` to help suggest this as a possible solution.

**Comment by DaveWM** [comment]
Thanks very much! Sorry it took me so long to respond to your issue, I only just noticed it :man_facepalming:. I've added a note in the readme about this problem, and also added a try/catch with a nicer exception message (available in version `0.2.0`).

**Comment by wbeard** [comment]
Thanks, @DaveWM!
